iree/hal/local/loaders/legacy_library_loader.cc

```diff
@@ -81,6 +81,13 @@ static iree_status_t iree_hal_dylib_executable_flatbuffer_verify(
                             "executable library_embedded is missing/empty");
   }
 
+  if (iree_DyLibExecutableDef_sanitized_kind_get(executable_def) ==
```

Comment:
If a new sanitizer kind is added this would fail to catch it and continue trying to load the executable. Instead, you can check to see if the sanitizer kind is set and if so require that the sanitizer is available (a switch statement would work) and otherwise return unavailable.

Reply:
But I added the SanitizerKind to flatbuffer in LLVMAOTTarget.cpp using
iree_DyLibExecutable_sanitized_kind(builder, options_.sanitizerKind);
And, I thought it can check the value of sanitizerKind with this condition.. Was it different from what you meant?
Thanks

Reply:
We need a switch statement. You will get a value from `iree_DyLibExecutableDef_sanitized_kind_get`, and we will have different check depending on the value.

If it is None, then nothing to check. If it is `address`, then we have to check ASAN features. If it s `memory`, we have to check MSAN features, etc. In this context, having a switch statement would be good.